README.md

```diff
@@ -14,3 +14,12 @@ More Lerna [commands](https://github.com/lerna/lerna#commands). Please use `yarn
 | [`react-ui-ag`](/packages/react-ui-ag) | [![npm](https://img.shields.io/npm/v/@rentpath/react-ui-ag.svg)](https://www.npmjs.com/package/@rentpath/react-ui-ag) | Apartmentguide.com specific components|
 | [`react-ui-tracking`](/packages/react-ui-tracking) | [![npm](https://img.shields.io/npm/v/@rentpath/react-ui-tracking.svg)](https://www.npmjs.com/package/@rentpath/react-ui-tracking) | Tracking HOCs and components|
 | [`react-ui-utils`](/packages/react-ui-utils) | [![npm](https://img.shields.io/npm/v/@rentpath/react-ui-utils.svg)](https://www.npmjs.com/package/@rentpath/react-ui-utils) | Utililty functions used to make things easier in all repos|
+
+## Working Locally
+
+Follow these steps to work with your local files instead of the published versions:
+
+1. cd into the react-ui package root folder, e.g. `packages/react-ui-core`
+1. run `yarn && yarn build`. You'll need to do this each time a file is changed.
+1. run `yarn link` in `packages/react-ui-core`
+1. in your project's root folder, run `yarn link @rentpath/react-ui-core`
```

Comment:
If possible use numbers 1. 2. 3. 4. on the lines to make the README file more readable as plain text?

Reply:
Are there any steps needed to "unlink" a project when you are done testing local files and want to return to the published versions?